dev/PushNotifications/PushNotificationsLongRunningTask/NotificationListener.cpp

```diff
@@ -22,26 +21,21 @@ STDMETHODIMP_(HRESULT __stdcall) NotificationListener::OnRawNotificationReceived
 
     if (!m_foregroundSinkManager->InvokeForegroundHandlers(m_appId, payloadArray, payloadLength))
     {
-        // Command line format: ----WindowsAppRuntimePushServer:-Payload:"<payloadAsEscapedUriFormat>"
-        std::wstring commandLine = L"----WindowsAppRuntimePushServer:-Payload:\"";
+        std::string commandLine = "----WindowsAppRuntimePushServer:-Payload:\"";
+        commandLine.append(reinterpret_cast<char*>(payload), payloadLength);
```

Comment:
Why are we re-introducing this escape character bug? @danielayala94 ?

Reply:
Saw something like this in the other PR as well.

Reply:
@pmpurifoy Please look at the existing code in WNP_LRP. This seems to be an incorrect merge.